scripts/scripts/sentence_to_fill_in_the_blanks.py

```
@@ -0,0 +1,120 @@
+#!/usr/bin/env python
+
+from typing import Dict, List, Any
```

Ruff (0.8.2)

`typing.Any` imported but unused

Remove unused import: `typing.Any`

(F401)